src/modal/modal.js

```diff
@@ -56,6 +57,21 @@ class Modal extends React.Component<ModalPropsT, ModalStateT> {
 
   componentDidMount() {
     this.setState({mounted: true});
+    if (__DEV__) {
+      if (!this.props.unstable_ModalBackdropScroll) {
```

Comment:
should we add a few words on what this property does?

Reply:
the is a description in the types and yard